### ISSUE TITLE
Color Contrast issue with "should"'s

### ISSUE THREAD

**Issue by clapierre** [w3c/epub-tests#101]
In the current table view of all the issues the Must issues all look fine, but the "Should" issues seem like they are grayed out which makes it hard to read and fails WCAG 2.4.3 for normal text size fonts.

Since we have the column for "Req" containing Must/Should etc. we don't really have to also visually gray out these rows.
Now perhaps just the words "Must" "Should" etc could had a different background

Black on Red - Must
Black on Magenta - Should
Black on Yellow - May
as shown here:
![Screen Shot: Text Must/Should/May with color backgrounds](https://user-images.githubusercontent.com/7736749/146228492-f4620967-5d64-441a-8b69-128effc890ba.png)
